jpyinterpreter/src/main/java/org/optaplanner/jpyinterpreter/util/JumpUtils.java

```diff
@@ -4,6 +4,10 @@
 import org.optaplanner.jpyinterpreter.PythonVersion;
 
 public class JumpUtils {
```

Comment:
Utility classes should have a private constructor.
Don't let people do things they shouldn't do, because they will.